README.md

```diff
@@ -1 +1,79 @@
 # Advanced Algorithms, Project 1 - Network Flow and Circulation with Demands Problem #
+
+Demonstrate the solution to the Ford-Fulkerson algorithm using
+Breadth-First Search to determine the shortest path between the
+source and sink nodes. Using the same algorithm, solve the
+Circulation problem.
+
+---
+
+## Building ##
+**NOTE**: The google test library is pulled down during the first build automatically. Be sure you are connected to the internet before your first build.
+
+### On Linux ###
+In a terminal within the project directory:
+
+```bash
+mkdir build # create a build folder in the project directory
+cd build # navigate into that build folder
+cmake .. # run cmake to generate makefiles
+make -j4 # build using 4 concurrent threads (increase the number if desired) 
+```
+
+### On Windows ###
```

Comment:
For the sake of brevity, I think we can safely omit Windows instructions.

Reply:
I honestly don't know what system our program will be tested on. Maybe we should ask in class, but it could very well be run on a Windows computer.